src/WorkBC.Tests/Tests/SearchTests.cs

```diff
@@ -203,6 +204,19 @@ public async Task SearchJobsExcludingPlacementAgencyJobs()
             //There should be results
             Assert.True(result.Count > 0, "Jobs excluding agency jobs did not return results");
         }
+        
+        [Theory(DisplayName = "When both salary string and salary numeric fields populated, calculate SalarySummary")]
+        [InlineData(39879152, "N/A")] // below minimum wage
+        [InlineData(40098629, "N/A")] // below minimum wage
```

Comment:
```
[InlineData(39879152, "N/A")] // below minimum wage
[InlineData(40098629, "N/A")] // below minimum wage
```
We currently have data protections in place to prevent showing salaries that are seem unrealisticly too high or too low.  I've asked the business if they want to keep these protections in place.